motioneye/static/js/main.js

```diff
@@ -2919,15 +2919,24 @@ function runPictureDialog(entries, pos, mediaType) {
     
     var img = $('<img class="picture-dialog-content">');
     content.append(img);
-    
+
+    var video_container = $('<video class="picture-dialog-content" controls="true" type="video/mp4">');
+    video_container.on('error', function(err) {
+      showErrorMessage('Error: Unable to play movie.')
```

Comment:
`err` has error code, that will be much useful to understand exact problem. Here is a list of errors and messages: https://gist.github.com/wilmoore/3252894. We can handle it in next PR.